backend/common/user-access.js

```diff
@@ -84,17 +82,17 @@ async function getUserDetails(userId) {
 
     let isBlocked = false;
     try {
-        const blocklist = await twitchApi.users.getAllBlockedUsersPaginated(streamerData.userId);
+        const blocklist = await twitchApi.getClient().helix.users.getBlocksPaginated(streamerData.userId).getAll();
         if (blocklist) {
-            isBlocked = blocklist.find(id => id === twitchUser.id) != null;
+            isBlocked = blocklist.find(user => user.userId === twitchUser.id) != null;
```

Comment:
since we just want a boolean, instead of `.find` we can use `.some`:
```js
isBlocked = blocklist.some(u => u.userId === twitchUser.id);
```

Reply:
Fixed.